packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.base.tsx

```diff
@@ -85,8 +85,8 @@ export class TeachingBubbleBase extends BaseComponent<ITeachingBubbleProps, ITea
       <Callout
         target={target || targetElement}
         onDismiss={onDismiss}
-        {...calloutProps}
         className={classNames.root}
+        {...calloutProps}
```

Comment:
wouldn't it be better to mix the calloutProps.className in with the className above?

Reply:
That makes sense, I'll make the change. I think the rest of the PR is still valid though.

Reply:
Actually, this is already done in line 79, where you send `stylesProps` to the `getClassNames` function, so that `calloutProps.className` is already part of `classNames.root`.